### PR TITLE
CON-1358 Feature add an apprentice

### PR DESCRIPTION
**Pull request by ChrisJWoodcock**

### CODE REVIEW

File: src/SFA.DAS.EmployerAccounts.Web/ViewModels/Row1Panel1ViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using SFA.DAS.EmployerAccounts.Web.Extensions;
+using System;
+
+namespace SFA.DAS.EmployerAccounts.Web.ViewModels
+{
+    public class Row1Panel1ViewModel
+    {
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerAccounts.Web/ViewModels/Row1Panel1ViewModel.cs]: Does this need to be merged with the call to action view model now?

File: src/SFA.DAS.EmployerAccounts/DependencyResolution/EncodingRegistry.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using SFA.DAS.Encoding;
+using StructureMap;
+
+namespace SFA.DAS.EmployerAccounts.DependencyResolution
+{
+    public class EncodingRegistry : Registry
+    {
+        public EncodingRegistry()
+        {
+            For<IEncodingService>().Use<EncodingService>().Singleton();
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerAccounts/DependencyResolution/EncodingRegistry.cs]: We may not want to mix usage of the EncodingService and HashingService

File: src/SFA.DAS.EmployerAccounts.Web/ViewModels/Row1Panel1ViewModel.cs (hunk 1 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.EmployerAccounts.Web/ViewModels/Row1Panel1ViewModel.cs]: Yes - but that is in the 1261 branch currently

File: src/SFA.DAS.EmployerAccounts/DependencyResolution/EncodingRegistry.cs (hunk 2 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.EmployerAccounts/DependencyResolution/EncodingRegistry.cs]: This one is for you@VasanthaKasirajan3008

Comment (VasanthaKasirajan3008) [reply src/SFA.DAS.EmployerAccounts/DependencyResolution/EncodingRegistry.cs]: We need to use Encoding Service to consume Commitments Api.

**Comment by cofaulco** [comment]
Draft PR other related PR's all now merged

**Pull request closed without merging**